### ISSUE TITLE
Post to slack whenever someone manually merges something into kubernetes/kubernetes

### ISSUE THREAD

**Issue by fejta** [kubernetes/test-infra#3026]
I like a prow plugin that allows us to specify:
* Users allowed to merge code (k8s-merge-bot in our case)
* Slack channels to broadcast (sig-contribex and kubernetes-dev)

Then whenever a user who is not on the whitelist merges something into the repo we send a slack message to those channels.

Example of a:
Submit queue merge: https://github.com/kubernetes/kubernetes/pull/45773#event-1118790064
Merge by a human: https://github.com/kubernetes/kubernetes/pull/46813#event-1116483350

In other words when Tim merged 46813 we want to see a note in kubernetes-dev and sig-contribex that says something to the effect of `Warning: @thockin manually merged https://github.com/kubernetes/kubernetes/pull/46813`

**Comment by spiffxp** [comment]
in an effort to herd submit-queue related discussion to one place, I think #sig-release would also probably be a good channel, at least during code freeze if nothing else

**Comment by ixdy** [comment]
it'd also be good if it could alert if someone pushes commits into the upstream branches directly (without going through a PR at all).
